custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "orderby",
+            "description": "Format: orderby=fieldName:asc or orderby=fieldName:desc. Applicable for all fields.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "select",
+            "description": "The 'select' parameter is used to retrieve only the specified fields from the collection. Format: select=id,name,accountNumber.",
+            "type": "string"
           },
           {
-            "name": "body",
             "in": "body",
-            "required": false,
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CustomerIds"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
             "schema": {
+              "required": [
+                "data"
+              ],
               "type": "object",
               "properties": {
-                "participants": {
-                  "title": "Participants",
+                "data": {
                   "type": "array",
                   "items": {
-                    "type": "object",
-                    "properties": {
-                      "communicationIdentifier": {
-                        "$ref": "#/definitions/CommunicationIdentifier"
-                      },
-                      "displayName": {
-                        "type": "string",
-                        "title": "Display Name",
-                        "description": "Display name for the chat participant."
-                      },
-                      "shareHistoryTime": {
-                        "type": "string",
-                        "title": "Time",
-                        "description": "Time in RFC3339 format (yyyy-MM-ddTHH:mm:ssZ) from which the chat history is shared."
-                      }
-                    }
-                  },
-                  "description": "Participants to add."
+                    "$ref": "#/definitions/Customer"
+                  }
                 }
               }
             }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        ]
+        }
       }
     },
-    "/chat/threads/{chatThreadId}/participants/:remove": {
-      "post": {
-        "responses": {
-          "204": {
-            "description": "Request successful"
-          },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
-          }
-        },
-        "summary": "Remove chat thread participant",
-        "description": "Remove participant from chat thread.",
-        "operationId": "RemoveParticipant",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/RemoveParticipant

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.